### PR TITLE
fix: App soft crash when deleting servers

### PR DESCRIPTION
**Pull request by ryanalexander**
Closes https://github.com/revoltchat/revite/issues/447

### CODE REVIEW

File: src/context/intermediate/modals/Prompt.tsx [hunk 1]
```diff
@@ -157,6 +158,7 @@ export const SpecialPromptModal = observer((props: SpecialProps) => {
                                         case "delete_channel":
```
Comment (Rexogamer) [inline src/context/intermediate/modals/Prompt.tsx:158]: I don't think redirecting to home is the best idea for channel deletions - probably best to split this into its own category

Comment (ryanalexander) [reply src/context/intermediate/modals/Prompt.tsx:158]: Honestly didn't even realise that delete channel was there lmao
I'll put a diff catch in there for that

**Comment by ryanalexander** [comment]
@Rexogamer new mechanic look better?

**Review by insertish — APPROVED** [review]
Changed path from `/home` to `/` since that is the path we should be redirecting to for home tab.

**Pull request merged**